website/docs/r/synthetics_monitor_step.html.markdown

```diff
@@ -39,7 +39,7 @@ The following are the common arguments supported for `STEP` monitor:
 * `account_id`- (Optional) The account in which the Synthetics monitor will be created.
 * `name` - (Required) The name for the monitor.
 * `uri` - (Required) The uri the monitor runs against.
-* `locations_public` - (Required) The location the monitor will run from. Valid public locations are https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips/. Exactly one of `locations_public` or `locations_private` is required.
+* `locations_public` - (Required) The location the monitor will run from. Valid public locations are https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips/. You don't need the `AWS_` prefix as the provider uses NerdGraph. At least one of either `locations_public` or `location_private` is required.
```

Comment:
Slight wording variation here, from exactly to "At least one", which aligns itself with the other four resources. 

Is the change in wording okay?

Reply:
That's a good catch! It should be "At least one of"